src/tree/azure/grouping/GroupingItem.ts

```diff
@@ -125,7 +125,8 @@ export class GroupingItem implements ResourceGroupsItem {
                     } catch (e) {
```

Comment:
Doesn't have to be this PR but:
* We should log this error.
* And make sure it gets reported to telemetry.

Reply:
Oddly, this command doesn't pass in a context. It's a slightly bigger change to start pushing in a context everywhere for error telemetry, so I'll just log to output channel for now.